QuizApp/User/UserView.swift

```diff
@@ -0,0 +1,72 @@
+import SwiftUI
+import Factory
+
+struct UserView: View {
+
+    @ObservedObject var viewModel: UserViewModel
+
+    var body: some View {
+        ZStack {
+            LinearGradient(gradient:
+                            Gradient(colors:
+                                        [Color(red: 0.453, green: 0.308, blue: 0.637),
+                                         Color(red: 0.154, green: 0.185, blue: 0.463)]),
+                           startPoint: .top,
+                           endPoint: .bottom)
```

Comment:
If you'll be using it through the app, you can create your custom view and just use it here.

Reply:
It is changed in the next PR (#32 )